src/Misc.ino

```diff
@@ -2343,20 +2361,35 @@ boolean conditionMatch(String& check)
 
   switch (compare)
   {
-    case '>':
-      if (Value1 > Value2)
-        match = true;
-      break;
-
-    case '<':
-      if (Value1 < Value2)
-        match = true;
-      break;
-
-    case '=':
-      if (Value1 == Value2)
-        match = true;
-      break;
+  case '>'+'=':
```

Comment:
Hmm, did not know this was possible in C++.
How does it work, since compare is just a char?

Usually you would need a "nextchar" or some kind of peek for this.

Reply:
it only add ascii codes to not use 2 chars for comparision

Reply:
they even can be out of range it is not a problem, but
ascii codes used here are under 0x40 so max value of sum is under 0x80 ;)